int32  position_command
int32  velocity_command
